### PR TITLE
Azure Orbital CLI extension

### PR DESCRIPTION
**Pull request by thisisdevanshu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Orbital

File: src/orbital/azext_orbital/azext_metadata.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/azext_metadata.json:2]: Do you need to release this extension in experimental? recommend to mark this extension isPreview first

Comment (thisisdevanshu) [reply src/orbital/azext_orbital/azext_metadata.json:2]: We are releasing the extension for our GA. What should we mark the extension in that case?

Comment (kairu-ms) [reply src/orbital/azext_orbital/azext_metadata.json:2]: > We are releasing the extension for our GA. What should we mark the extension in that case?

In that case, you can remove this line.

File: src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital contact-profile create",
+)
+class Create(AAZCommand):
+    """Creates or updates a contact profile.
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py:18]:
```suggestion
    """Create a contact profile.
```

File: src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+
+    :example: Create or Update Contact Profile
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py:20]:
```suggestion
    :example: Create a Contact Profile
```

File: src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_delete.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital contact-profile delete",
+    confirmation="Are you sure you want to perform this operation?",
+)
+class Delete(AAZCommand):
+    """Deletes a specified contact profile resource.
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_delete.py:19]:
```suggestion
    """Delete a specified contact profile resource.
```

File: src/orbital/azext_orbital/aaz/latest/orbital/spacecraft/contact/_create.py [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital spacecraft contact create",
+)
+class Create(AAZCommand):
+    """Creates a contact.
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/spacecraft/contact/_create.py:18]:
```suggestion
    """Create a contact.
```

File: src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py (hunk 2 above)
Comment (thisisdevanshu) [reply src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py:18]: yeah the PUT API behind this cmdlet does both create and update.

Comment (thisisdevanshu) [reply src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py:18]: All the summaries are from our public docs https://docs.microsoft.com/en-us/rest/api/orbital/earthobservation/contact-profiles/create-or-update?tabs=HTTP

File: src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_update.py [hunk 6]
```diff
@@ -0,0 +1,691 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital contact-profile update",
+)
+class Update(AAZCommand):
+    """Update contact profile tags.
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_update.py:18]: Why tags?
```suggestion
    """Update contact profile.
```

File: src/orbital/azext_orbital/aaz/latest/orbital/spacecraft/_update.py [hunk 7]
```diff
@@ -0,0 +1,493 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital spacecraft update",
+)
+class Update(AAZCommand):
+    """Update spacecraft tags.
```
Comment (kairu-ms) [inline src/orbital/azext_orbital/aaz/latest/orbital/spacecraft/_update.py:18]:
```suggestion
    """Update spacecraft.
```

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**